src/jwplayer/players/MobileArticleVideoPlayer/mobileArticleVideoPlayer.module.scss

```diff
@@ -2,13 +2,15 @@
 	width: 100%;
 	height: 56.25vw;
 	position: relative;
+    z-index: 100;
```

Comment:
todo: seems prettier didn't work here?

Reply:
this seems to be a github bug. it looks differently in IDE